src/nspawn/nspawn-mount.c

```diff
@@ -1066,7 +1066,7 @@ static int setup_volatile_yes(const char *directory, uid_t uid_shift, const char
         if (!mkdtemp(template))
                 return log_error_errno(errno, "Failed to create temporary directory: %m");
 
-        options = "mode=755";
+        options = "mode=755,size=10%,nr_inodes=400k";
```

Comment:
should be 25%, as this includes /var, too, and thus needs to be at least as large as we assign /var